joomla/joomla-cms
# [4.0] Add json to source_formats of template editor

### Testing Instructions
- Go to `administrator/index.php?option=com_templates&view=templates`
- Click on "Cassiopeia Details and Files"
- See file list on the left:
File `joomla.asset.json` (an essential file for templates configuration in J4) is missing.
